source/_UIAHandler.py

```diff
@@ -154,6 +132,9 @@
 	UIA_SystemAlertEventId:"UIA_systemAlert",
 }
 
+if winVersion.isAtLeastWin10():

```

Comment:
While I really like this new winVersion helper function, I think it might be better to tie this to a minimum version of the UIAutomation object. This also makes sure that it works in source copies of NVDA. If I'm correct, Windows 10 started with UIAutomation4, but @josephsl may no more.

I think we could also reregister for UIA_Text_TextSelectionChangedEvent while at it.

Reply:
As much as linking this to UIAutomation interfaces would make this work for source copies,  it doesn't quite fit logically, nor is there necessarily an interface for the exact release version we want to match against.